src/hotspot/share/oops/klassVtable.cpp

```diff
@@ -452,7 +471,7 @@ bool klassVtable::update_inherited_vtable(InstanceKlass* klass, const methodHand
     // This is safe because the method at this slot should never get invoked.
     // (TBD: put in a method to throw NoSuchMethodError if this slot is ever used.)
     if (super_method->name() == name && super_method->signature() == signature &&
-        (!_klass->is_interface() ||
+        (!klass->is_interface() ||
```

Comment:
Is _klass and klass really the same? I thought they could be different. If so, this looks like a subtle behavioural change hidden in a large cleanup patch.

Reply:
Yes, they are the same which is why I made this change.  I thought of taking out the klass parameter and using _klass everywhere so there isn't this extra _klass v. klass distinction to worry about.

Reply:
Thanks it is more clear in the new patch.